src/app/util/types_stub.h

```diff
@@ -1934,4 +1934,39 @@ typedef struct
 #define emberAfPluginColorControlServerComputePwmFromHsvCallback(endpoint) (void) 0
 #define emberAfPluginColorControlServerComputePwmFromTempCallback(endpoint) (void) 0
 
+/**
+ * @brief Macro that copies the token value from non-volatile storage into a RAM
+ * location.  This macro can only be used with tokens that are defined using
+ * DEFINE_INDEXED_TOKEN.
+ *
+ * @note To better understand the parameters of this macro, refer to the
+ *           example of token usage above.
+ *
+ * @param data   A pointer to where the token data should be placed.
+ *
+ * @param token  The token name used in <code>DEFINE_*_TOKEN</code>,
+ *               prepended with <code>TOKEN_</code>.
+ * @param index  The index to access in the indexed token.
+ */
+#define halCommonGetIndexedToken(data, token, index)
```

Comment:
these have to plumb to persistent storage or be replaced by CHIP config code?

Reply:
Definitely want an issue tracking implementing this (and the "set" version) for real and link to it here.

Reply:
https://github.com/project-chip/connectedhomeip/issues/2470
Using this one for this as well.